### ISSUE TITLE
A11y_AzureDataStudio_AddNewController_ScreenReader : NVDA is not announcing state of the selected list item for "Authentication Type" combo box.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7603]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#WCAG4.1.2;DesktopApp,#NVDA;#JAWS;#SQL Azure Data Studio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA , JAWS

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Connect to the servers with the required credentials.
3. Navigate to the "SQL Server Big Data Cluster" and expand it.
4. Navigate to the "SQL Server Big Data Cluster controller" and click on it.
5. Start NVDA.
6. Navigate till "Authentication Type " combo box using tab.
7. Observe the announcement by screen reader.

**Actual:**
NVDA is not announcing state of the selected list item for "Authentication Type" combo box.
"Basic " is selected list item but screen reader only announce "basic combo box basic collapsed " 
**Similar issue is also observed with JAWS .**

**Expected:**
NVDA is not announcing state of the selected list item for "Authentication Type" combo box.
For example , if "Basic " is selected list item then screen reader should announce "basic combo box basic selected collapsed " 

**User Impact:**
Screen reader user will find it difficult to understand that whether the navigated  item is selected or not.

**Attachment for Reference:**
[7603_A11y_AzureDataStudio_AddNewController_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3707500/7603_A11y_AzureDataStudio_AddNewController_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
@12-shweta I'm going to close this since this seems to be the default behavior for select controls and none of the other dropdowns have had this issue raised against them. Let me know if you have information about this being an expected standard.

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7603_Closed](https://user-images.githubusercontent.com/54656883/72725174-1c644080-3bab-11ea-92a8-644c543f90f9.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
